lib/iris/__init__.py

```diff
@@ -471,3 +472,14 @@ def sample_data_path(*path_to_join):
             "appropriate for general file access.".format(target)
         )
     return target
+
+
+def use_plugin(plugin_name):
```

Comment:
This needs including in [`__all__`](https://github.com/SciTools/iris/blob/b08cfa63bb435566546d9deb52985ea4561cf391/lib/iris/__init__.py#L118-L132) before it will be listed in the documentation.